src/Components/Dropdown/Dropdown.css

```diff
@@ -10,7 +10,7 @@ select {
   text-align: center;
```

Comment:
Removing the border here and adding a little bit of color might help this dropdown look more modern